client/src/components/ForumPostContainer/ForumPostContainer.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./ForumPostContainer.css";
+import ForumPost from "../ForumPost/ForumPost";
+import Comment from '../Comment/Comment'
+
+export default function ForumPostContainer(props) {
+  return (
+    <div className="PostContainer">
+      <div>
+        {props.forumPost ? (
+          <ForumPost
+            pillClick={props.forumPost.pillClick}
+            commentClick={props.forumPost.commentClick}
+            likeClick={props.forumPost.likeClick}
+            to={props.forumPost.to}
+            name={props.forumPost.name}
+            likes={props.forumPost.likes}
+            comments={props.forumPost.comments}
+            topics={props.forumPost.topics}
+            image={props.forumPost.image}
+            post={props.forumPost.post}
+          />
+        ) : (
+          <ForumPost />
+        )}
+      </div>
+      <div>
+        {props.comment ? (
```

Comment:
It's possible that there may be multiple comments on a post. So maybe `props.comment` should be `props.comments` (i.e. an array of comments) and then this becomes a loop with `.map()` or something?